compiler/onecc-docker/onecc-docker.py

```diff
@@ -0,0 +1,64 @@
+import sys
```

Comment:
Format checking before commiting the codes would make an easier review.

You can check it by a command `./nnas format` or `./nnas format --diff-only`.

Reply:
Oh, thank you for the code review. I'm not sure what you mean by part `./nnas format` or part `./nnas format --diff-only`. Is there anything I can refer to?

Reply:
I mean `/ONE/nnas`:)